### DISCUSSION TITLE
UEFI bootloader for Dunfell

### DISCUSSION THREAD

**Discussion by andregirard37** [OE4T/meta-tegra#1291] [Q&A]
Hello,
I believe Jetson Linux R32.7.3 and R32.7.4 (JetPack 4.6.3 and 4.6.4) are supporting the UEFI bootloader. From https://developer.nvidia.com/embedded/linux-tegra-r3274 . Yet, when building Dunfell with the latest update for 4.6.4 (I was seeing the same for 4.6.3), I still see the cboot being supported. When building with Kirkstone, UEFI bootloader is supported which makes sense. I couldn't find any discussion here on why that is that Dunfell still supports cboot.
Thanks!

**Comment by madisongh** [comment]
The opening sentence on the page you linked to looks like generic, boilerplate text about Jetson Linux that gets inserted into every sub-page.  However, that text was apparently updated for Jetson Linux R35.x, which is where UEFI replaced cboot, without considering that it would appear at the head of the pages for all the older releases as well.

There is no UEFI support in the R32 series.
